src/json/members/jaredmit

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Jared Mitovich",
+  "pennkey": "jaredmit",
+  "major": "Communication: Data nad Network Science",
+  "school": "College",
+  "bio": "Hi! I'm Jared. I'm a sophomore from New Jersey studying Communication with minors in Data Science and Consumer Psychology.",
+  "team": "Penn Mobile",
+  "roles": ["Business Developer"],
+  "photo": "https://imgur.com/a/E7hRHqm",
```

Comment:
I think you copied the wrong photo URL. Try right clicking the image itself and selecting "copy image address".